Modules/Core/Common/include/itkMultiThreaderBase.h

```diff
@@ -173,28 +189,32 @@ class ITKCommon_EXPORT MultiThreaderBase : public Object
    * (void *)arg passed into the SetSingleMethod.
    *
    * DEPRECATED! Use WorkUnitInfo instead. */
-ITK_GCC_PRAGMA_DIAG_PUSH()
-ITK_GCC_PRAGMA_DIAG(ignored "-Wattributes")
-INTEL_PRAGMA_WARN_PUSH
-INTEL_SUPPRESS_warning_1292
-CLANG_PRAGMA_PUSH
-CLANG_SUPPRESS_Wc__14_extensions
-#ifdef ITK_LEGACY_SILENT
-  struct ThreadInfoStruct
-#else
-  struct [[deprecated( "Use WorkUnitInfo, ThreadInfoStruct is deprecated since ITK 5.0" )]] ThreadInfoStruct
-#endif
-CLANG_PRAGMA_POP
-INTEL_PRAGMA_WARN_POP
+  ITK_GCC_PRAGMA_DIAG_PUSH()
+  ITK_GCC_PRAGMA_DIAG( ignored "-Wattributes" )
+  INTEL_PRAGMA_WARN_PUSH
+  INTEL_SUPPRESS_warning_1292 CLANG_PRAGMA_PUSH CLANG_SUPPRESS_Wc__14_extensions
```

Comment:
This change definitely looks wrong.

Reply:
What looks wrong about this?  It looks fine to me.

Reply:
Don't macros each need a line of its own? If not, why do preceding macros each get a line all to themselves?

Reply:
Ahh, I see.  This may be a rare case where we need
// clang-format off

// clang-format on

-- The macros that are now nested are defined as empty
```
#  define INTEL_SUPPRESS_warning_1292
#  define CLANG_PRAGMA_PUSH
#  define CLANG_PRAGMA_POP
#  define CLANG_SUPPRESS_Wfloat_equal
#  define CLANG_SUPPRESS_Wc__14_extensions
```
On this platform, so they are can be nested for this preprocessor options.